src/ashley/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -1085,12 +1086,16 @@ msgid "Can archive a forum"
 msgstr "Peut archiver un forum"
 
 #: machina_extensions/forum_permission/defaults.py:27
+msgid "Can manage a user's moderator status"
+msgstr "Permet de gérer le statut de modérateur d'un utilisateur"
```

Comment:
I checked in Django and confirmed that permission names were not translated. After digging a little my understanding is that this name is destined to go to a model field that is not translatable anyway... https://github.com/django/django/blob/main/django/contrib/auth/models.py#L62